src/Promitor.Core.Scraping/Configuration/Model/Metrics/ScrapeDefinitionBatch.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using Promitor.Core.Contracts;
+
+namespace Promitor.Core.Scraping.Configuration.Model.Metrics
+{
+    /// <summary>
+    /// Defines properties of a batch of scrape definitions 
+    /// </summary>
+    public class ScrapeDefinitionBatchProperties : IEquatable<ScrapeDefinitionBatchProperties>
+    { 
+
+    }
+
+    /// <summary>
+    /// Configuration about the Azure Monitor metric to scrape
+    /// </summary>
+    public AzureMetricConfiguration AzureMetricConfiguration { get; }
+
+
+    /// <summary>
+    /// The scraping model.
+    /// </summary>
+    public Scraping Scraping { get; }
+
+    /// <summary>
+    /// The Azure subscription to get the metric from. This should be used instead of using
+    /// the SubscriptionId from <see cref="Resource"/> because this property will contain
+    /// the global subscription id if none is overridden at the resource level.
+    /// </summary>
+    public string SubscriptionId { get; }
+
+    /// <summary>
+    /// The Azure resource type shared by all scrape definitions in the batch
+    /// </summary>
+    public ResourceType ResourceType { get; }
+
+    public TimeSpan AggregationInterval{ get; }
+
+    public override int GetHashCode()
```

Comment:
Do you think this is a sane way to implement HashCode?

Reply:
Do we need one? If not, let's remove it

Reply:
The idea I have is to use hash code computation to enforce batch restrictions. This part should be sane. 

How I'm choosing to implement hash code is hashing a unique string representation of `ScrapeDefinitionBatch`. Curious what you think/can think of something better?

Reply:
If we really need one, then this is OK for me